### ISSUE TITLE
No test suites running

### ISSUE THREAD

**Issue by ettmetal** [freeCodeCamp/testable-projects-fcc#1077]
#### Issue Description

Clicking 'Run Tests' for any of the test suites provided by the Fork Me! FCC template does not run the tests. Instead, the content "#mocha div missing, add it to your document" is added to the page.

#### Browser Information

- Browser Name, Version: Firefox, 74.0
- Operating System: Windows 10
- Mobile, Desktop, or Tablet: Desktop

#### Your Code / Link to Your Pen

So far, I've found that this is reproducable from the [Fork Me! FCC pen](1), but my survey form pen where I first noticed the issue is at https://codepen.io/ettmetal/pen/wvarKjb?editors=1100

#### Screenshot

Running any test suite on the Fork Me! pen results in this:

![screenshot showing the addition of #mocha div missing, add it to your document after running a test suite](https://user-images.githubusercontent.com/3324453/78377565-0a9e1100-75d0-11ea-93de-983d434c99d5.png)

I'd expect the behaviour to be that the tests run, and fail.

[1]: https://codepen.io/freeCodeCamp/pen/MJjpwO

**Comment by petercr** [comment]
@ettmetal I opened up the link to your CodePen. All you needed to do was select which test you wanted to run from the drop down menu. And then all the tests run and pass correctly.

I have attached a screenshot for reference below: ⬇️ 
Please go ahead and close this issue. Also make sure to try to get help in the Free Code Camp Forum **_before_** opening an issue. 
Thanks

### Screenshot
![passing tests](https://user-images.githubusercontent.com/26460352/78414834-1bf51680-75ec-11ea-87ea-968033dfb909.png)

**Comment by ettmetal** [comment]
@petercr - I did select a test suite, in fact I tried to run all of them. Sorry if that was not clear. Running any suite results in the message about a missing #mocha div, and no tests are run.

**Comment by petercr** [comment]
The screenshot is of your CodePen. I opened it up and ran the tests. They all ran and passed. 17/17 
There was no error when I ran them

**Comment by Ryuno-Ki** [comment]
Perhaps a script blocker intervened? I had some trouble with mine (uMatrix) as well.

**Comment by ettmetal** [comment]
Sorry to have left it so long.

I looked into script blocking - I don't have that enabled. Firefox does however have pretty strict CORS policies, and it looks like codepen's server isn't configured to send the correct headers to enable that.

So I looked into other browsers. Chrome and Vivaldi are both showing the same behaviour, with Javascript enabled, and I had the same results (the mocha div message), including in incognito mode (no extensions). There are several errors in the console for each browser. Vivaldi is also reporting that it's unable to access a resource due to CORS policy, whereas Chrome shows `uncaught DOMException: failed to read the 'localstorage' property...` a couple times (no permission was requested).

It still looks like it might be a configuration issue, so I'll take it to the forums. It looked like a bug to me initially, sorry for that.
